src/index.js

```diff
@@ -20,40 +26,45 @@ class Timer extends EventEmitter {
     this._pauseTime = 0
     this._duration = null
     this._timeoutID = null
-    this._status = 'stopped' // 'running' or 'paused'
+    this._status = status._stopped // 'running' or 'paused'
   }
 
   start (duration, interval) {
-    if (this._status !== 'stopped') return
+    if (this._status !== status._stopped) return
     if (duration == null) throw new TypeError('must provide duration parameter')
     this._duration = duration
     this._endTime = Date.now() + duration
-    this._status = 'running'
+    this.changeStatus(status._running)
     this.emit('tick', this._stopwatch ? 0 : this._duration)
     this._timeoutID = setInterval(tick.bind(this), interval || this._interval)
   }
 
   stop () {
     clearInterval(this._timeoutID)
-    this._status = 'stopped'
+    this.changeStatus(status._stopped)
   }
 
   pause () {
-    if (this._status !== 'running') return
+    if (this._status !== status._running) return
     this._pauseTime = Date.now()
-    this._status = 'paused'
+    this.changeStatus(status._paused)
   }
 
   resume () {
-    if (this._status !== 'paused') return
+    if (this._status !== status._paused) return
     this._endTime += Date.now() - this._pauseTime
     this._pauseTime = 0
-    this._status = 'running'
+    this.changeStatus(status._running)
+  }
+  
+  changeStatus (status) {
+    this._status = status
+    this.emit('onChangeStatus', this._status)
```

Comment:
I would like to rename the event, so it is attached by `timer.on('statusChanged', ...)`
```suggestion
    this.emit('statusChanged', this._status)
```

Reply:
I updated as per your suggestions.

Thanks!